Project/Project/CDTTodoReplicator.h

```diff
@@ -18,8 +18,8 @@
 
 #import <CloudantSync.h>
 
-@interface CDTTodoReplicator : NSObject<CDTReplicatorDelegate>
+@interface CDTTodoReplicator : NSObject<CDTReplicatorDelegate, NSURLSessionConfigurationDelegate>
 
--(void)sync;
+- (void)syncInBackgroundWithCompletionHandler:(void (^)())completionHandler;
```

Comment:
Since we've added the completion handler why don't we use it in the AppDelegate?